library/pnotes.inc.php

```diff
@@ -34,8 +34,12 @@ function getPnoteById($id, $cols = "*")
  */
 function checkPnotesNoteId(int $id, string $user): bool
 {
-    $check = sqlQuery("SELECT `id` FROM pnotes WHERE id = ? AND user = ? AND deleted != 1", array($id, $user));
-    if (!empty($check['id']) && ($check['id'] == $id)) {
+    $check = sqlQuery("SELECT `id`, `user`, `assigned_to` FROM pnotes WHERE id = ? AND deleted != 1", array($id));
```

Comment:
While this works I'd gone with 
$check = sqlQuery("SELECT `id` FROM pnotes WHERE id = ? AND (user = ? || assigned_to = ?) AND deleted != 1", array($id, $user, $user));

I'm just going to merge yours and pull into my quest. okay